docs/diagrams/ViewActivityDiagram.puml

```diff
@@ -0,0 +1,32 @@
+@startuml
+
+start
+
+if () then ([inputs are valid))
+    switch ()
+    case ([entry type is income])
+      :Retrieve the list of income;
+    case ([entry type is expenditure])
+      :Retrieve the list of expenditure;
+    case ([else])
+      :Display an error message;
+      stop
+    endswitch
+
+    switch ()
+    case ([category graph type is specified])
```

Comment:
Are there checks for presence of `mo/` param when graph type is `month`. Should we add this in the activity diagram?

Reply:
I think the level of abstraction is quite good for the activity diagram.